src/main/java/it/pagopa/transactions/client/PaymentGatewayClient.java

```diff
@@ -170,7 +169,7 @@ public Mono<VposAuthResponseDto> requestCreditCardAuthorization(AuthorizationReq
                                         .holder(cardData.getHolderName())
                                         .securitycode(cardData.getCvv())
                                         .isFirstPayment(true) // TODO TO BE CHECKED
-                                        .threeDsData(cardData.getThreeDsData())
+                                        .threeDsData(new String(Base64.getDecoder().decode(cardData.getThreeDsData())))
```

Comment:
Use here the String constructor with explicit encoding, the used one here use the system default one. 
Better to use here the explicit expected encoding such as UTF-8 to avoid problems with special characters encoding